lib/evm_database_ops.rb

```diff
@@ -190,4 +190,19 @@ def self.backup_file_name(action = :backup)
     "#{action == :backup ? BACKUP_TMP_FILE : DUMP_TMP_FILE}_#{time_suffix}"
   end
   private_class_method :backup_file_name
+
+  FILE_SPLITTER = File.expand_path("../manageiq/util/file_splitter.rb", __FILE__).freeze
+  def self.pg_admin_opts(db_opts, additional_opts)
+    if additional_opts[:byte_count]
+      split_opts = [
+        [:byte_count, additional_opts[:byte_count]],
+        [nil, "-"],
+        [nil, db_opts[:local_file]]
```

Comment:
This can be simplified to 

```ruby
      split_opts = [
        additional_opts.slice(:byte_count),
        "-",
        db_opts[:local_file]
      ]
```

Reply:
Oh, I was under the impression that it had to be a fully associative array.  I will look into doing this.